src/backend/distributed/metadata/metadata_cache.c

```diff
@@ -2898,6 +2899,24 @@ JsonbExtractPathTextFuncId(void)
 }
 
 
+/*
+ * CitusDependentObjectFuncId returns oid of the is_citus_depended_object function.
+ */
+Oid
+CitusDependentObjectFuncId(void)
+{
+	if (MetadataCache.CitusDependentObjectFuncId == InvalidOid)
```

Comment:
Given that `is_citus_depended_object` is only created on tests with GUC enabled, maybe we can make it explicit here as well:

```C
if (! HideCitusDependentObjects)
{
ereport(ERROR, "is_citus_depended_object can only be used while running the regression tests");
}